src/components/QueryOutput/QueryOutput.js

```diff
@@ -1829,8 +1829,8 @@ export class QueryOutput extends React.Component {
           }
 
           // No logical operators detected, just compare numbers
-          const number = parseFloat(rowValue?.replace(/[^0-9.]/g, ''))
-          const filterNumber = parseFloat(headerValue?.replace(/[^0-9.]/g, ''))
+          const number = parseFloat(rowValue?.toString().replace(/[^0-9.]/g, ''))
+          const filterNumber = parseFloat(headerValue?.toString().replace(/[^0-9.]/g, ''))
```

Comment:
This would also strip out any negative signs from the number? Can we guarantee that a negative value wouldn't be here?